test/e2e/test_tools.py

```diff
@@ -4,25 +4,27 @@
 can be called with the basic configuration that is available via CLI. More thorough testing of each method is done in
 separate unit tests/e2e tests.
 """
+import copy
+import datetime
 import json
-import unittest
 import os
-import datetime
 import re
+import unittest
+
 import jsonpath_ng
 import jsonpath_ng.ext
-import copy
 
 from knora.dsplib.utils.excel_to_json_lists import excel2lists, validate_lists_section_with_schema
+from knora.dsplib.utils.excel_to_json_project import excel2project
 from knora.dsplib.utils.excel_to_json_properties import excel2properties
 from knora.dsplib.utils.excel_to_json_resources import excel2resources
 from knora.dsplib.utils.id_to_iri import id_to_iri
-from knora.dsplib.utils.onto_create_ontology import create_project
-from knora.dsplib.utils.onto_validate import validate_project
 from knora.dsplib.utils.onto_create_lists import create_lists
+from knora.dsplib.utils.onto_create_ontology import create_project
 from knora.dsplib.utils.onto_get import get_ontology
-from knora.dsplib.utils.xml_upload import xml_upload
+from knora.dsplib.utils.onto_validate import validate_project
 from knora.dsplib.utils.shared import validate_xml_against_schema
+from knora.dsplib.utils.xml_upload import xml_upload
 from knora.excel2xml import excel2xml
```

Comment:
what tool does this import sorting? would be nice for the diffs, if those would only show up once (i.e. always the same automation)